src/main/drivers/bus_spi.h

```diff
@@ -56,7 +56,7 @@ typedef enum {
 #elif defined(STM32F7)
     SPI_CLOCK_SLOW          = 256, //00.42188 MHz
     SPI_CLOCK_STANDARD      = 16,  //06.57500 MHz
-    SPI_CLOCK_FAST          = 4,   //27.00000 MHz
+    SPI_CLOCK_FAST          = 8,   //13.50000 MHz
```

Comment:
This needs some consideration.

Is `SPI_CLOCK_FAST` only used for gyros atm?

Reply:
as per slack conversation, no SD card, flash etc.